includes/class-wc-payments-woopay-button-handler.php

```diff
@@ -333,6 +333,7 @@ public function display_woopay_button_html() {
 		}
 
 		$settings = $this->get_button_settings();
+		$radius   = WC_Payments_Features::is_stripe_ece_enabled() ? $settings['radius'] : 10;
```

Comment:
When ECE is disabled that implies that the Payment Request Buttons are enabled instead. Those do not support changing the border radius, so we need a fallback value when ECE is disabled.

A default radius of 10px was something I just chose arbitrarily while working on the PR. We'll probably want some other default radius for when ECE is disabled, likely to match what the PRBs use.